linux/src/threadpool_linux.c

```diff
@@ -598,7 +641,6 @@ int threadpool_timer_start(THREADPOOL_HANDLE threadpool, uint32_t start_delay_ms
                     LogError("Failure calling timer_delete. Error: %d: (%s)", errno, err_msg);
                 }
             }
-            free(timer_instance);
```

Comment:
>  free(timer_instance); [](http://example.com/codeflow?start=10&length=23)

we can leak timers in the ENABLED state on the error path here now

Reply:
No the timers never need to be freed since it's just an array and not malloced.